### PR TITLE
Add cert name in renewal error messages (#4932)

### PR DESCRIPTION
**Pull request by r5d**
Update `certbot.renewal.handle_renewal_request` to print cert name in error messages.

### CODE REVIEW

**Comment by jsha** [comment]
@ohemorange can you take a look since you worked on the lineage stuff?

**Comment by ohemorange** [comment]
What's the intended goal of this PR? The cert name is by definition the renewal config filename minus the `.conf`. Were you looking to add the domain names covered by the cert?

**Comment by jsha** [comment]
See my reasoning in https://github.com/certbot/certbot/issues/4932. I wanted to delete a certificate because it was failing renewal. IMO it's not immediately clear that the certificate name is part of the filename.

File: certbot/renewal.py [hunk 1]
```diff
@@ -389,14 +389,16 @@ def handle_renewal_request(config):
...
         # Note that this modifies config (to add back the configuration
         # elements from within the renewal configuration file).
         try:
             renewal_candidate = _reconstitute(lineage_config, renewal_file)
         except Exception as e:  # pylint: disable=broad-except
-            logger.warning("Renewal configuration file %s produced an "
-                           "unexpected error: %s. Skipping.", renewal_file, e)
+            logger.warning("Renewal configuration file %s (cert: %s) "
+                           "produced an unexpected error: %s. Skipping.",
+                           lineagename, renewal_file, e)
```
Comment (ohemorange) [inline certbot/renewal.py]: `lineagename` and `renewal_file` should be reversed

**Comment by ohemorange** [comment]
Ah, that makes sense!

File: certbot/renewal.py (hunk 1 above)
Comment (r5d) [inline certbot/renewal.py]: Done.

**Pull request merged**